src/types.d.ts

```diff
@@ -71,3 +71,14 @@ declare interface Comment {
   };
   children: Comment[];
 }
+
+declare interface Reactions {
+  current_user: {
+    id: null;
+  };
+  reactions: [];
```

Comment:
Even tho we don't have authentication of any sort now, it's good to provide the full typings in case we add it some day.
```suggestion
    id: string | null;
  };
  reactions: {
    category: string;
    created_at: string;
    id: number;
    points: number;
    reactable_id: number;
    reactable_type: string;
    status: string;
    updated_at: string;
    user_id: number;
  }[];
```